src/map/entities/charentity.cpp

```diff
@@ -2191,7 +2191,7 @@ void CCharEntity::OnItemFinish(CItemState& state, action_t& action)
         actionTarget.knockback = 0;
     }
 
-    if ((isParalyzed && scrollProtection && isScroll && itemLoss) || (isParalyzed && !itemLoss)) // Become paralyzed and stop executing.
+    if ((isParalyzed && scrollProtection && PItem->isScroll() && itemLoss) || (isParalyzed && !itemLoss)) // Become paralyzed and stop executing.
```

Comment:
This looks overall good.  The only thing I would change is put in
```
bool isScroll = PItem->getID();
```
and then leave the line at 2194 untouched from the original.  Functionally the same, but easier to read.

Reply:
Done.